train.py

```diff
@@ -178,6 +181,22 @@ def run_train(
     optimizer = get_optimizer(model=model, cfg=cfg)
     scheduler = get_scheduler(cfg=cfg, optimizer=optimizer, epoch_steps=epoch_steps)
 
+    if cfg.environment.use_deepspeed:
+        (
+            model,
+            optimizer,
+            train_dataloader,
+            val_dataloader,
+            scheduler,
+        ) = deepspeed_initialize(
+            model=model,
+            optimizer=optimizer,
+            lr_scheduler=scheduler,
+            training_data=train_dataloader.dataset,
+            validating_data=val_dataloader.dataset,
+            cfg=cfg,
+        )
+
```

Comment:
should that maybe go to the `wrap_model_distributed` function to reduce if/else checks?

Reply:
[d1efef5](https://github.com/h2oai/h2o-llmstudio/pull/288/commits/d1efef56fd2a7de4de01dff4a1b511696294803c)

update here